libc-test/semver/illumos.txt

```diff
@@ -14,8 +14,13 @@ POSIX_FADV_NORMAL
 POSIX_FADV_RANDOM
 POSIX_FADV_SEQUENTIAL
 POSIX_FADV_WILLNEED
+POSIX_SPAWN_SETSID
 posix_fadvise
 posix_fallocate
+posix_spawn_file_actions_addchdir
+posix_spawn_file_actions_addchdir_np
+posix_spawn_file_actions_addfchdir
```

Comment:
Solaris 11.4 does have these three too:
```
$ nm /usr/lib/libc.so.1  | grep posix_spawn_file_actions_addchdir
001c5b20 T posix_spawn_file_actions_addchdir
001c5b20 W posix_spawn_file_actions_addchdir_np
$ nm /usr/lib/libc.so.1  | grep posix_spawn_file_actions_addfchdir
001c5a20 T posix_spawn_file_actions_addfchdir
```
The same definitions. It would be nice to have them in solatish.txt.

Reply:
Thanks, but not `addfchdir_np` it seems?

Reply:
Added these.